src/modules/cmdpal/ext/Microsoft.CmdPal.Ext.Indexer/Pages/IndexerPage.cs

```diff
@@ -29,6 +29,13 @@ public IndexerPage()
         PlaceholderText = Resources.Indexer_PlaceholderText;
         _searchEngine = new();
         _queryCookie = 10;
+
+        EmptyContent = new CommandItem(new NoOpCommand())
+        {
+            Icon = Icon,
+            Title = Resources.Indexer_Subtitle,
+            Subtitle = Resources.Indexer_NoResultsMessage + "\n\n" + Resources.Indexer_NoResultsMessageTip,
```

Comment:
Should we only include this `Subtitle` when the user has actually typed something? Seems weird to say "no results found" if the user hasn't actually typed something

Reply:
@zadjii-msft
I need your feedback here. Do you only want that for file explorer as WW shows a list of all open windows without query?